packages/odyssey/src/scss/base/_iconography.scss

```diff
@@ -1,6 +1,6 @@
 .ods-icon {
   position: relative;
-  top: -0.1em;
+  top: -0.05em;
```

Comment:
Why did this change?

Reply:
As we were putting together Dismiss, @joejezowski-okta noticed that the icons still weren't seated properly. I consider this an interim fix - we should revisit Icon entirely after we update the type again.